src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -32,21 +33,36 @@ namespace triton { namespace perfanalyzer { namespace clientbackend {
 namespace tritonremote {
 //==============================================================================
 
+struct triton::client::HttpSslOptions ParseHttpSslOptions(
+    const struct triton::perfanalyzer::clientbackend::SslOptionsBase&);
+std::pair<bool, struct triton::client::SslOptions> ParseGrpcSslOptions(
+    const struct triton::perfanalyzer::clientbackend::SslOptionsBase&);
+
```

Comment:
You don't need to define functions here. 
can you do 
```
namespace {
    triton::client::HttpSslOptions ParseHttpSslOptions(
    const triton::perfanalyzer::clientbackend::SslOptionsBase&) {
     ... your implementation ...
   }
std::pair<bool, triton::client::SslOptions> ParseGrpcSslOptions(
    const triton::perfanalyzer::clientbackend::SslOptionsBase&)  {
     ... your implementation ...
   }
}
```

Reply:
switched to this 👍